docs/cmd/finch_help.md

```diff
@@ -0,0 +1,14 @@
+# finch help
```

Comment:
Does not work properly as per https://github.com/runfinch/finch/issues/221

Reply:
Yeah, `run` is special cased to disable `-h`, not sure the best way to handle this. Might need to change something upstream to note that -h doesn't work on `run`.

https://github.com/containerd/nerdctl/blob/main/cmd/nerdctl/container_run.go#L77-L78

Reply:
Oh, got it. Cool, will keep tracking that issue then. Thanks!